.github/workflows/build.yml

```diff
@@ -21,21 +31,28 @@ jobs:
           submodules: true
 
       - name: Install LLVM (for running tests)
-        run: sudo apt-get -y install llvm
+        run: |
+          sudo apt-get -y install llvm-${{ env.LLVM_VERSION }}
+          # Inspired by https://gist.github.com/junkdog/70231d6953592cd6f27def59fe19e50d
+          sudo update-alternatives \
+            --install /usr/bin/llvm-config       llvm-config      /usr/bin/llvm-config-${{ env.LLVM_VERSION }} 100 \
```

Comment:
The `install` with priority 100 assumes that it's a higher priority than anything currently used, so that the `auto` mode will choose this llvm.  It might be better to follow this with a `sudo update-alternatives --set llvm-config /usr/bin/llvm-config-${{ env.LLVM_VERSION }}` (I think that's the right syntax... maybe use `update-alternatives --list llvm-config` to verify) to ensure this is the selected alternative.

Reply:
I think `--set` only makes sense when you have specified multiple alternatives to choose from. When I tried your suggestion in CI, [it fails with](https://github.com/GaloisInc/llvm-pretty-bc-parser/actions/runs/4440707184/jobs/7794849944?pr=205#step:3:25):

```
update-alternatives: error: no alternatives for llvm-as
```

At the very least, we can run `update-alternatives --list` on each tool to make sure that what we have `--install`ed is what we intended.

Reply:
Oh wait, this only applies to `llvm-config` itself, not anything listed after `--slave`...